### ISSUE TITLE
Plugin does not work when MythicMobs is installed

### ISSUE THREAD

**Issue by ArmonicPvP** [Nathat23/StackMob-5#175]
Mobs stacking does not work when MythicMobs is installed. The entire stack will die, but it will not respawn another mob.

**Comment by Nathat23** [comment]
any errors in console?

**Comment by ArmonicPvP** [comment]
No, but when I disable MythicMobs, it works fine. This is the latest dev build of MythicMobs.

**Comment by Nathat23** [comment]
Is this a new issue? Otherwise I have a problem as I can't access the dev builds of mythicmobs.

**Comment by ArmonicPvP** [comment]
I can test if it is. I can provide a copy of the dev version if you need.

**Comment by Nathat23** [comment]
can you test with the free builds?

**Comment by Nathat23** [comment]
I've just tested with the free builds any it seems to work fine.
